cadcutils/setup.cfg

```diff
@@ -31,8 +35,8 @@ url = http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca
 edit_on_github = False
 github_project = opencadc/cadctools 
 # version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
+install_requires = requests lxml>=3.7.0 six
```

Comment:
restricted to requests>=2.8,<=2.11.1. I've also removed this dependency from vos.